src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+interface ITodoModal {
+  setModal: (value: boolean) => void;
+  userId: number;
+  todo: Omit<Todo, 'userId'>;
+}
+
+const useUser = (userId: number) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getUser(userId)
+      .then(setUser)
+      .catch(e => alert(e))
```

Comment:
It's not the best idea to use `alert()`, because it blocks the whole browser interface.
Usually, in such cases some custom warning component is rendered. I'm not sure if it's the part of your task, so you can just display console message - like you do in `App.tsx`.